src/world_model/world_model.py

```diff
@@ -62,27 +62,26 @@ def default_policy(self, state):
         return None
 
     @lru_cache(maxsize=None)
-    def possible_successors(self, state, action=None, n_samples = None):
-        """Return a list of possible successor states after performing action in state,
+    def possible_successors(self, state, action=None, n_samples = None) -> set:
+        """Return the of possible successor states after performing action in state,
```

Comment:
typo, it should be "Return the set of ..."

Reply:
Fixed.